File: app/src/main/java/io/defy/chicken/lover/network/response/ChickenInfoRes.java
package io.defy.chicken.lover.network.response;

import com.google.gson.annotations.SerializedName;
import org.json.JSONArray;
import org.json.JSONObject;

import java.util.Date;
import java.util.List;

public class ChickenInfoRes {

    @SerializedName("way")
    String way;

    @SerializedName("name")
    String name;

    @SerializedName("brand")
    String brand;

    @SerializedName("type_array")
    String type_array;

    @SerializedName("type_number")
    int type_number;

    @SerializedName("thumbs_up")
    int thumbs_up;

    public String getWay() {
        return way;
    }

    public String getName() {
        return name;
    }

    public String getBrand() {
        return brand;
    }

    public String getType_array() {
        return type_array;
    }

    public int getType_number() { return type_number; }

    public int getThumbs_up() { return thumbs_up; }
}

File: app/src/main/java/io/defy/chicken/lover/network/response/ChickenInfoRes.kt
package io.defy.chicken.lover.network.response

import com.google.gson.annotations.SerializedName

data class ChickenInfoRes(@SerializedName("_id") var _id : Int = 0,
                          @SerializedName("way") var way : String,
                          @SerializedName("name") var name : String,
                          @SerializedName("brand") var brand : String,
                          @SerializedName("type_array") var type_array : String,
                          @SerializedName("type_number") var type_number : Int = 0,
                          @SerializedName("thumbs_up") var thumbs_up : Int = 0)